apps/docs/azure-functions/src/lib/auth.ts

```
@@ -54,64 +134,184 @@ export function getTokenClaims(request: HttpRequest): TokenClaims | null {
+ */
+function isAdminFromClaims(claims: TokenClaims): boolean {
+  if (!claims?.email) return false;
+  const domain = claims.email.split("@")[1]?.toLowerCase();
+  return ADMIN_DOMAINS.includes(domain);
+}
+
+/**
+ * Require authentication with proper JWT validation (async)
+ * This validates JWT signature using Azure AD B2C JWKS
+ */
+export async function requireAuthAsync(request: HttpRequest): Promise<{
```

ESLint

[error] Multiple exports of name 'requireAuthAsync'.

(import/export)